src/enum/HttpReponseStatusForSubscription.enum.ts

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * Http Response Status to be sent back to dapr runtime as subscription response, as per 
+ * https://docs.dapr.io/reference/api/pubsub_api/#expected-http-response
+ */
+export enum HttpResponseStatusForSubscription {
+    SUCCESS = "SUCCESS",  // Message is processed successfully
+    RETRY = "RETRY", // Message to be retried by Dapr
+    DROP = "DROP", // Warning is logged and message is dropped
+    Others = "Others" // Error, message to be retried by Dapr
```

Comment:
(1) "Others" is not a part of the contract, and (2) Drop/Others are not used by the SDK. Shall we remove both of them?

Reply:
I had kept them to keep in sync as per the docs. And, `Others` seem like a bit off, so definitely can remove. I am not sure about DROP, I just kept it for maintaining the sanctity of having all options available.